_data/simple-icons.json

```
@@ -1445,7 +1445,7 @@
         },
         {
             "title": "Backendless",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

```suggestion
            "hex": "5056E5",
```
They use this color a lot on their website. Might be a better choice, what do you think?